Testreglar/1.1.1/App/app-1.1.1a.json

```diff
@@ -40,129 +72,281 @@
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Er det mulig å sveipe til minst et bilde på appsiden?",
+			"ht": "<p>Sjekk om det er mulig å sveipe til minst ett av bildene du fant i forrige steg</p>\n<p>Slik tester du:</p>\n<ul>\n<li>aktiver skjermleser\n<ul>\n<li>iOS: VoiceOver</li>\n<li>Android: Talkback</li>\n</ul>\n</li>\n<li>åpne appsiden </li>\n<li>forsøk å sveipe til bildet</li>\n</ul>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Ikkje testbart",
+					"utfall": "Det er ikke mulig å sveipe til bilder på appsiden."
 				}
 			}
 		},
 		{
 			"stegnr": "3.1",
-			"spm": "Beskriv bildet som skal vurderast.",
-			"ht": "<p>Beskriv bildet med få stikkord.</p>",
+			"spm": "Hvilket bilde tester du?",
+			"ht": "<p>Beskriv bildet, slik at det er mulig å identifisere det i ettertid. Hvis det gjelder flere bilder, registrerer du et og et.</p>",
 			"type": "tekst",
-			"kilde": [],
-			"label": "Bilde:",
 			"ruting": {
 				"alle": {
 					"type": "gaaTil",
 					"steg": "3.2"
 				}
 			},
+			"label": "Bilde:",
 			"multilinje": true
 		},
 		{
 			"stegnr": "3.2",
-			"spm": "Er bildet meint å vere pynt/dekor?",
-			"ht": "<p>Eksempel på bilde som er pynt/dekor: usynlige bilde, gjennomsiktige bilde, pyntebilde og border. Slike bilde er meint å vere pynt og gir ikkje informasjon.</p>",
+			"spm": "Er bildet ren dekorasjon, brukes bildet utelukkende til visuell formattering eller er bildet ikke synlig for brukeren? ",
+			"ht": "<p>Ikoner og grafiske elementer som er lagt til for å fremheve det visuelle, kan ofte oppfattes som unødvendig \"støy\" eller forstyrrende for skjermleserbrukere (<a href=\"https://medialt.no/bildebeskrivelser/1481.aspx\" target=\"_blank\" rel=\"noopener\">MediaLT</a>).</p>\n<p>Sjekk om bildet</p>\n<ul>\n<li>er ren dekorasjon eller</li>\n<li>brukes utelukkende til visuell formatering eller</li>\n<li>ikke presenteres for brukeren</li>\n</ul>\n<p><strong>Ren dekorasjon:</strong> Begrepet \"<a href=\"https://www.w3.org/Translations/WCAG21-no/#dfn-pure-decoration\" target=\"_blank\" rel=\"noopener\">ren dekorasjon</a>\" er definert i WCAGs ordliste: <em>\"innhold som bare har et estetisk formål, som ikke inneholder informasjon eller funksjonalitet\"</em>. Likevel er det en skjønnsmessig vurdering hvilke bilder det gjelder. Definisjonen peker på to tilfeller:</p>\n<ul>\n<li>bildet har kun et estetisk formål</li>\n<li>bildet inneholder ikke informasjon eller funksjonalitet.</li>\n</ul>\n<p><strong>Utelukkende visuell formattering</strong></p>\n<p>Eksempel på bilde som brukes utelukkende til visuell formattering er en bord eller en virvel som brukes til å skape en visuell effekt.</p>\n<p><strong>Ikke viser (presenteres) for brukeren</strong></p>\n<p>Bilde som ikke vises til brukeren er usynlige bilder, som for eksempel brukes for å spore bruksstatistikk på en appside.</p>",
```

Comment:
Bør vi også ha med "Hvis bilder ikke gir noe mer informasjon enn teksten rundt"? Dette har vørt et issue i tilsynsarbeid ihvertfall.

Reply:
Vi har lagt inn et merknad for dette i både nett-1.1.1a og her i app-1.1.1a
> Merk: Der bildet ikke gir mer informasjon eller funksjonalitet enn løpende tekst som hører til bildet, er bildet vanligvis dekorativt